content/post/thriving-oss-community.md

```diff
@@ -0,0 +1,138 @@
++++
+title = "How to Build a Thriving Open-source Community"
+draft = false
+date = 2017-11-20T00:00:00Z
+tags = [
+    "wallaroo",
+    "community"
+]
+categories = [
+    "Community"
+]
+description = "In this post, I will share some of the initial ideas, tactics, and approaches that we are employing and believe will drive the success of our community."
```

Comment:
The style of this description feels a little different than the other ones we've had. I think maybe it could be shorted to "Some of the initial ideas, tactics, and approaches that we are employing and believe will drive the success of our community." to better fit with the others.

Reply:
Ill make this change.